resources.md

```diff
@@ -0,0 +1,62 @@
+# Useful resources, research, and quick links
+
+## Purpose  
+Serve as a quick resource for the sig to grab links that we work with at high frequency, reference, other orgs research. This is not official guidance. 
```

Comment:
This sentence seems awkward to me, and we should probably capitalize SIG. Maybe something like this ...
"Serve as a quick resource for the SIG to grab links that we work with at high frequency and reference from other research."